examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    plt.gca().set_prop_cycle("color", ["red", "green", "blue"])
</pre><p> </p><p>the colors used should be the ones of the theme, or alternatively, use <code>C1</code>,  <code>C3</code>,  <code>C0</code>  in plots to alter the order. red and green look the same to some colorblind people.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Not sure how to use colors from the theme so that I can match <code>data</code> and <code>pred</code> colors. Also, if I set more than one color in a plot, for examples with <code>plt.plot(t, y, ".", color = ("C0", "C1", "C3"))</code>, I get the error <code>Cannot put cycle reference ('C0') in prop_cycler</code>.</p>

Reply:
I see. I think the best way to go is calling plot on `data` directly, then use `plt.gca().set_prop_cycle(None)` to reset the color cycle and then call plot on `pred`. This will ensure both plots use c0, c1 and c2 which are the first 3 colors of the theme and a colorblind friendly color pallette. 

Extra nitpick, given that we use the axes object however, I personally would prefer using `ax.` instead of the `plt` interface.